app/build.gradle

```diff
@@ -270,24 +272,24 @@ dependencies {
     implementation("io.coil-kt:coil-compose:2.6.0")
```

Comment:
latest for Coil 2.7.0, but aiming to remove glide, probably makes sense to handle together in isolation cc @ycheng-kickstarter let me know if you agree on this piece :)